code/modules/debugging/tracy.dm

```diff
@@ -57,6 +57,18 @@ GLOBAL_REAL(Tracy, /datum/tracy)
 	return FALSE
 #endif
 
+/// Flushes the byond-tracy file, if the Paradise version of byond-tracy which outputs a file is used.
+/datum/tracy/proc/flush()
```

Comment:
Nothing calls this

Reply:
> although currently it's not called (it also gets flushed by the `destroy` call) - this is just for the sake of documenting that it exists in case anyone wants to use it in the future, altho if anyone knows good places to call it, let me know.

Reply:
Why not have the Destroy call this then?

Reply:
because call_ext("destroy") flushes anyways - this is in case a manual flush is ever needed.